assets/css/app.css

```diff
@@ -1,3 +1,6 @@
+@import "./markdown.css";
+@import "./code.css";
```

Comment:
These imports should be at the bottom to avoid conflicts with Tailwind, right?

Yet, if I move them after `@tailwind utilities;`, the styles in `markdown.css` and `code.css` are not loaded.

What could be wrong?

Reply:
@import tailwind does works:
```
@import "tailwindcss/base";
@import "tailwindcss/components";
@import "tailwindcss/utilities";

@import "./markdown.css";
@import "./code.css";
```

@tailwind doesn’t:
```
@tailwind base;
@tailwind components;
@tailwind utilities;

@import "./markdown.css";
@import "./code.css";
```